recipe/meta.yaml

```
@@ -8,9 +8,12 @@ package:
 source:
   url: https://pypi.io/packages/source/p/psutil/psutil-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    - 0001-Add-missing-includes-for-macOS.patch
+    - 0002-upstream-patch-for-pypy.patch  # [python_impl == 'pypy']
```

It's ok to apply this generally as upstream did as well

```suggestion
    - 0002-upstream-patch-for-pypy.patch
```